src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -219,11 +232,8 @@ def create(dataset, target, feature, max_iterations=10,
                                     label=train_data['labels'].to_numpy(),
                                     batch_size=training_batch_size, shuffle=True)
 
-    custom_NN = SoundClassifier._build_custom_neural_network(feature_extractor.output_length, num_labels, custom_layer_sizes)
-    ctx = _mxnet_utils.get_mxnet_context()
-    custom_NN.initialize(mx.init.Xavier(), ctx=ctx)
-
-    trainer = mx.gluon.Trainer(custom_NN.collect_params(), 'nag', {'learning_rate': 0.01, 'momentum': 0.9})
+    from ._mx_sound_classifier import SoundClassifierMXNetModel
+    mxnet_model = SoundClassifierMXNetModel(feature_extractor, num_labels, custom_layer_sizes, verbose)
```

Comment:
Why do we need to pass the entire feature extractor?

Reply:
Also, let's not call this `mxnet_model`. The entire point of this is that this doesn't have to be MXNet.

Reply:
Both good points. Changing now